.changeset/odd-maps-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/underscore-redirects': minor
```

Comment:
Since this package is v0, should this be a patch instead? Seems like it should be non-breaking.